src/xdocs/checks/coding/multiplestringliterals.xml

```diff
@@ -32,39 +32,25 @@
             </tr>
             <tr>
               <td>allowedDuplicates</td>
-              <td>
-                Specify the maximum number of occurrences to allow without generating a
-                warning.
-              </td>
+              <td>Specify the maximum number of occurrences to allow without generating a warning.</td>
               <td><a href="../../property_types.html#int">int</a></td>
               <td><code>1</code></td>
               <td>3.5</td>
             </tr>
+            <tr>
+              <td>ignoreOccurrenceContext</td>
+              <td>Specify token type names where duplicate strings are ignored even if they don't match ignoredStringsRegexp. This allows you to exclude syntactical contexts like annotations or static initializers from the check.</td>
+              <td><a href="../../property_types.html#subset of tokens TokenTypes">subset of tokens TokenTypes</a></td>
```

Comment:
Not a valid link.
https://checkstyle.org/property_types.html#subset%20of%20tokens%20TokenTypes

Reply:
Should be similar to https://checkstyle.org/checks/coding/finallocalvariable.html#FinalLocalVariable

Reply:
Bad view: https://checkstyle.org/checks/coding/multiplestringliterals.html#MultipleStringLiterals

Reply:
@rnveach , I removed update on this module from this PR, we will deal with MultipleStringLiterals separately

Reply:
moved to https://github.com/checkstyle/checkstyle/pull/13721

Reply:
Leaving this open for others to see the bug.